plugins/importtext_procedure.cc

```diff
@@ -797,39 +797,46 @@ struct ImportTextProcedureWorkInstance
 		          SqlCsvScope& scope)
 	{
 
-		const size_t numberOutputColumns = columnNames.size();
-
+    const bool outputColumnNames = !areOutputColumnNamesKnown;
```

Comment:
There seems to also be an indentation problem here making the diff look bigger than it is actually.